x-pack/plugins/observability/public/components/app/section/apm/index.tsx

```diff
@@ -21,29 +21,34 @@ import { StyledStat } from '../../styled_stat';
 import { onBrushEnd } from '../helper';
 
 interface Props {
-  startTime?: string;
-  endTime?: string;
+  absoluteTime: { start?: number; end?: number };
+  relativeTime: { start: string; end: string };
   bucketSize?: string;
 }
 
 function formatTpm(value?: number) {
   return numeral(value).format('0.00a');
 }
 
-export const APMSection = ({ startTime, endTime, bucketSize }: Props) => {
+export const APMSection = ({ absoluteTime, relativeTime, bucketSize }: Props) => {
   const theme = useContext(ThemeContext);
   const history = useHistory();
 
+  const { start, end } = absoluteTime;
   const { data, status } = useFetcher(() => {
-    if (startTime && endTime && bucketSize) {
-      return getDataHandler('apm')?.fetchData({ startTime, endTime, bucketSize });
+    if (start && end && bucketSize) {
+      return getDataHandler('apm')?.fetchData({
+        absoluteTime: { start, end },
+        relativeTime,
+        bucketSize,
+      });
     }
```

Comment:
I _think_ you can avoid the conditional because all three should be required.

```suggestion
      return getDataHandler('apm')?.fetchData({
        absoluteTime: { start, end },
        relativeTime,
        bucketSize,
      });
```

Reply:
Explanation why https://github.com/elastic/kibana/pull/71259#discussion_r452744232